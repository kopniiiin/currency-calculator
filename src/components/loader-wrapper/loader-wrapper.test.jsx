import React from "react";
import {shallow} from "enzyme";

import LoaderWrapper from "./loader-wrapper.jsx";

describe(`snapshot test: LoaderWrapper`, () => {
  it(`should render correctly`, () => expect(
    shallow(<LoaderWrapper>ðŸ¦¥</LoaderWrapper>)
  ).toMatchSnapshot());
});
