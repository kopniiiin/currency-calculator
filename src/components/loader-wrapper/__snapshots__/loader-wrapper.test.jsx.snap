// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot test: LoaderWrapper should render correctly 1`] = `
<div
  className=" loader-wrapper"
>
  ðŸ¦¥
</div>
`;
